roxie/ccd/ccdmain.cpp

```diff
@@ -717,6 +717,20 @@ int STARTQUERY_API start_query(int argc, const char *argv[])
         if (udpSnifferEnabled && !roxieMulticastEnabled)
             DBGLOG("WARNING: ignoring udpSnifferEnabled setting as multicast not enabled");
 
+        int ttlTmp = topology->getPropInt("@multicastTTL", 1);
```

Comment:
Very minor, but is it cleaner to do -

int ttlTmp = topology->getPropInt("@multicastTTL", multicastTTL);

so that the default value is set in one place instead of 2 (here and where it is declared) ?